### ISSUE TITLE
Some autoloaded functions are no longer found as of 2022-08-22

### ISSUE THREAD

**Issue by grettke** [org2blog/org2blog#323]
Unfortunately I had to make a change that stomped on some autoloads that were renamed. If you are using them you're going to get an error and they won't work anymore. Normally we don't make breaking API changes without a big version number change. However this is an exception because although there is a technical way to deal with plain old renames this is not a plain old rename, it is an autoloads thing that changed in a newer version of Emacs.

I forgot these long ago and must deal with it now because the upgraded package manager is failing to install Org2Blog correctly on some computers. Although I'm hoping that you've already bound your keys to the updated functions long ago I recognize that is an unfair assumption. My apologies for the change: it won't happen again with a big version number change and even bigger notifications.

Here are the functions that got stomped on along with their replacement.

- Replace `org2blog/wp-login` with `org2blog-user-login`
- Replace `org2blog/wp-new-entry` with `org2blog-buffer-new`
- Replace `org2blog/wp-post-subtree` with `org2blog-subtree-post-save`
- Replace `org2blog/wp-preview-buffer-post` with `org2blog-buffer-post-or-page-view`
- Replace `org2blog/wp-preview-subtree-post` with `org2blog-subtree-post-or-page-view`
- Replace `org2blog/wp-track-buffer` with `org2blog-buffer-track`
- Replace `org2blog/wp-track-subtree` with `org2blog-subtree-track`

**Comment by grettke** [comment]
@punchagan The reason I made this change is [here](https://github.com/org2blog/org2blog#how-does-org2blog-manage-autoloads). The reason for the _possible_ breaking is that I renamed all of those functions and used aliases to redirect to them. Consequently the autoloads for these 7 functions are gone. Judgement call is that it is more realistic to make the change and have users read this issue. This is an exception and in the future breaking API changes will almost always include a major version number change.
